modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -21,8 +20,14 @@ kind: NUMAResourcesScheduler
 metadata:
   name: numaresourcesscheduler
 spec:
-  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}"
+  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}" <1>
 ----
++
+<1> In a disconnected environment, make sure to configure the resolution of this image by either:
```

Comment:
[Lead-in sentences should stand alone rather than continuing to the bullets
](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording)
```suggestion
<1> In a disconnected environment, make sure to configure the resolution of this image by completing one of the following actions:
```

Reply:
I didn't review this PR. I just wanted to add that comment, similar to the comment I made in the 4.16 CP PR. Sorry for the confusion.

Reply:
no biggie! was just guessing based on comment history :clinking_glasses: